Makefile

```diff
@@ -229,16 +235,19 @@ CLI_BINARY_NAME := broker-cli
 build-mac-broker-cli: check-docker-compose
 	docker-compose run --rm -e GOOS=darwin -e GOARCH=amd64 dev \
 		go build -o ${CONTRIB_BINARY_DIR}/${CLI_BINARY_NAME} \
+		-ldflags '$(LDFLAGS)' \
```

Comment:
Not sure we need to set this on the CLI builds.

Reply:
Still think we don't need these flags here.

Reply:
Even though the CLI is not supposed to be a first-class citizen, it couldn't hurt, right?

Reply:
I'm fine with it. Should this PR include the necessary modification to `contrib/cmd/cli`?

Reply:
Repeating what @krancour and I talked about offline: I am going to remove this and submit a low-priority issue to:

- Add the flag back
- Implement the `version` command